### DISCUSSION TITLE
[Question]: Best way to upgrade when using docker-compose?

### DISCUSSION THREAD

**Discussion by donvito** [danny-avila/LibreChat#4554] [Q&A]
### What is your question?

Hi, 

Just wanted to ask. What is the best way to upgrade when using docker-compose? I just deleted the ghcr.io/danny-avila/librechat-dev image in my local docker and it seemed to work. 

But, for production use, how can I guarantee that all dependencies are upgraded like database migrations and other dependencies which are in other docker containers?

How can I keep the user and messages data during migration?

Thank you.

### More Details

No other details

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
This: https://www.librechat.ai/docs/local/docker#update-librechat

You won't loose your data

**Reply by frdteknikelektro** [reply]
I've already run this but the version on the bottom still not changing

**Reply by fuegovic** [reply]
might be cached in your browser, try clearing the cache and/or accessing in a private session
